source/aggregation-pipeline-builder.txt

```diff
@@ -159,6 +159,25 @@ To view and change pipeline settings:
 
 .. _pipeline-custom-collation:
 
+Explain Plan for Pipelines
+--------------------------
+
+Use the :guilabel:`Explain Plan` to help you understand the performance of 
```

Comment:
Perhaps "Use the :guilabel:`Explain Plan` panel ..."?
Not sure what the convention is for naming in the GUI. Perhaps examine how other similar items are named?

Reply:
Done.